qa/standalone/mon/mon-stretched-cluster.sh

```diff
@@ -143,7 +143,140 @@ EOF
 
     sleep 3
 
-    teardown $dir || return 1
```

Comment:
This is by mistake and caused the test to fail in teuthology, we should tear down first.